src/main/java/com/devonfw/quarkus/general/service/exception/mapper/NotFoundExceptionMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.devonfw.quarkus.general.service.exception.mapper;
+
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class NotFoundExceptionMapper extends AbstractExceptionMapper implements ExceptionMapper<NotFoundException> {
```

Comment:
I do not see, why we create an individual mapper class for each exception type.
I would suggest to implement `[Response]ExceptionMapper<Throwable>` once and then simply check for specific exception types.
IMHO your code is harder to get an overview of what is going on for which exception type. Otherwise I would have all in one place and the code is more or less the same everywhere. The only specific aspect is to determine things like status code what will be done in a if block...

Reply:
The problem is that by using `ExceptionMapper<Throwable>` the exception mapper of [RESTEasy](https://github.com/quarkusio/quarkus/tree/main/extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime) are used because the are more specific than our mapper. To "override" them we have to implement the exception mapper for each type like for example NotFoundException or UnauthorizedException directly. I think this is also adressed with this issue https://github.com/quarkusio/quarkus/issues/7883

Reply:
Thanks for the explanation. Can you then add this valuable info (including link) to the JavaDoc of `AbstractExceptionMapper`

Reply:
@hohwille added the information